CHANGE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v3.2.3
+- Improved the stability and reliability of the plugin.
```

Comment:
Actual changes to be released are included in the not-yet merged PRs #1395 and #1396